include/renderers/gl/mir/renderer/gl/buffer_render_target.h

```diff
@@ -36,79 +34,14 @@ class WriteMappableBuffer;
 }
 namespace gl
 {
-class Context;
-
-template<void (*allocator)(GLsizei, GLuint*), void (* deleter)(GLsizei, GLuint const*)>
-class GLHandle
-{
-public:
-    GLHandle()
-    {
-        (*allocator)(1, &id);
-    }
-
-    ~GLHandle()
-    {
-        if (id)
-            (*deleter)(1, &id);
-    }
-
-    GLHandle(GLHandle&& from)
-        : id{from.id}
-    {
-        from.id = 0;
-    }
-
-    operator GLuint() const
-    {
-        return id;
-    }
-
-private:
-    GLuint id;
-};
-
-using RenderbufferHandle = GLHandle<&glGenRenderbuffers, &glDeleteRenderbuffers>;
-using FramebufferHandle = GLHandle<&glGenFramebuffers, &glDeleteFramebuffers>;
 
 /// Not threadsafe, do not use concurrently
 class BufferRenderTarget: public RenderTarget
 {
```

Comment:
This is an API and ABI change: we need to bump MIRSERVER_ABI. (Yes, that isn't very intuitive.)

...and bumping MIRSERVER_ABI means consolidating the symbol.map.

Reply:
There may be other ABI breaks, but buffer_render_target.h is introduced in this PR, so I don't think removing things from it is an ABI break.